bundles/core/org.eclipse.smarthome.core/src/main/java/org/eclipse/smarthome/core/internal/i18n/ResourceBundleTracker.java

```diff
@@ -30,16 +31,15 @@
  * @author Markus Rathgeb - Add locale provider support
  * @author Ana Dimova - fragments support
  */
-@SuppressWarnings("deprecation")
-public class ResourceBundleTracker extends ResolvedBundleTracker {
+@SuppressWarnings({ "deprecation", "rawtypes", "unchecked" })
```

Comment:
The only "solution" I found has been https://github.com/maggu2810/smarthome/commit/b7785d751d10e9f0fb1444a3ebfa194f206d074b which combines the suppress warnings to the sections that really need it.
But if it is necessary to create separate classes... I assume you will tell me, that it isn't :wink: